app/components/primer/alpha/overlay/footer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Primer
+  module Alpha
+    class Overlay
+      # A `Overlay::Footer` is a compositional component, used to render the
+      # Footer of an overlay. See <%= link_to_component(Primer::Alpha::Overlay) %>.
+      class Footer < Primer::Component
+        # @param show_divider [Boolean] Show a divider between the footer and body.
+        # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
+        def initialize(
+          show_divider: false,
```

Comment:
Footer is missing an align_content param as detailed here https://github.com/github/primer/blob/main/apis/overlay-api.md#overlayfooter

```
.Overlay-footer.Overlay-footer--alignStart
.Overlay-footer.Overlay-footer--alignCenter 
.Overlay-footer.Overlay-footer--alignEnd
```